src/js/Templates/Shared/Image.js

```diff
@@ -67,6 +70,12 @@ export default class Image extends React.Component {
     }
 
     onError(event) {
+        const state = store.getState();
+        const activeApp = state.activeApp;
+        if (activeApp) {
+            const fileName = path.basename(this.props.image);
+            UIController.onUpdateFile(activeApp, fileName);
+        }
```

Comment:
Oddly I only get this notification for my `appIcon`, even if I send a nonexistant image in an `AddSubMenu` request

Reply:
I am not seeing that issue. Can you post the message you are sending?

Here is the android code i was using for an addSubMenu with no image:


```
                AddSubMenu as = new AddSubMenu();
                as.setMenuName("sub");
                as.setMenuID(70);
                Image i = new Image();
                i.setImageType(ImageType.DYNAMIC);
                i.setValue("fake_file.png");
                as.setMenuIcon(i);
                sdlManager.sendRPC(as);
```

I only receive the onerror for the missing submenu icon when viewing the menu: 
![Screen Shot 2020-08-05 at 12 15 36 PM](https://user-images.githubusercontent.com/12038416/89437313-68c13900-d715-11ea-95ac-48286009c39f.png)

Do you see an onerror in the console for the app icon?